SocialCareCaseViewerApi/V1/UseCase/PersonalRelationshipsUseCase.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections.Generic;
+using SocialCareCaseViewerApi.V1.Boundary.Requests;
+using SocialCareCaseViewerApi.V1.Exceptions;
+using SocialCareCaseViewerApi.V1.Gateways;
+using SocialCareCaseViewerApi.V1.UseCase.Interfaces;
+
+namespace SocialCareCaseViewerApi.V1.UseCase
+{
+    public class PersonalRelationshipsUseCase : IPersonalRelationshipsUseCase
+    {
+        private IDatabaseGateway _databaseGateway;
```

Comment:
can be `readonly`

Reply:
I need to get VS Code to pick these up! 🤦